src/configureWorkspace/configurePython.ts

```diff
@@ -66,6 +66,16 @@ gunicorn==20.0.4`;
 const flaskRequirements = `flask==1.1.1
 gunicorn==20.0.4`;
 
+const lowRightsUser = `
+# Switching to a low-rights user, please refer to https://aka.ms/vscode-docker-python-user
+RUN useradd appuser && chown -R appuser /app
+USER appuser
+`;
+
+const rootUserWarning = `
+# Warning: A port below 1024 has been exposed. This requires the image to run as a root user which is not a best practice.
+# For more information, please refer to https://aka.ms/vscode-docker-python-ports`
```

Comment:
Can't we have just one link?

Reply:
I think one link is fair, I'll check back with Uche since he's the one who advised on the wording/links.

Reply:
I would change the link then to 'please refer to https://aka.ms/vscode-docker-python-user-rights`